src/avrouting.cs

```diff
@@ -51,6 +51,21 @@ interface AVCustomRoutingActionItem {
 		string OverrideTitle { get; set; }
 	}
 
+	[NoWatch, NoTV, NoMac, iOS (16, 1)]
```

Comment:
Does this need MacCatalyst? Looks like MacCatalyst is supported from the AVCustomRoutingController interface below